src/event_handler.rs

```diff
@@ -697,6 +697,10 @@ impl EventHandler {
 }
 
 fn is_remap(actions: &Vec<KeymapAction>) -> bool {
+    if actions.len() == 0 {
```

Comment:
Makes sense, but the meaning of this code wasn't immediately clear to me. Could you leave a comment above this to clarify how this works and what this is for?

Reply:
Sure, I have rebased, and added a commit with an explanation.